source/plugin/commands/childcommands.rst

```diff
@@ -55,7 +55,7 @@ If a command has child commands, a :javadoc:`CommandExecutor`, set through
 :javadoc:`Builder#executor(CommandExecutor)` and the parameters for the executor are set to optional. 
```

Comment:
The only potential grammar issue I could find is with this wording: 

> If a command has child commands, a `CommandExecutor`, set through `Builder#executor(CommandExecutor)` and the parameters for the executor are set to optional.

I'm unsure what is meant by this?

Reply:
Hi. This means that under normal circumstances, the ``executor`` method is a required method to be called when building a command. However when the command has children, it is not required. 

Essentially its a requirement to either set a executor or to set children of the command.

Hope that helps. 

Thanks for helping out the docs :)

Reply:
@mosemister how do the new changes look? If the before is preferred, this change can be reverted. Everything else regarding spelling and grammar/flow looks good to me--I can't identify any other issues with this page.

Reply:
Looks good. Approved